webapp/app/forms/steps/lotse_multistep_flow_steps/personal_data_steps.py

```diff
@@ -40,15 +40,15 @@ class Form(SteuerlotseBaseForm):
             validators=())
         familienstand_married_lived_separated = YesNoField(
             label=_l('form.lotse.familienstand_married_lived_separated'),
-            render_kw={'example_input': _l('form.lotse.familienstand_married_lived_separated.example_input'),
+            render_kw={'data-example-input': _l('form.lotse.familienstand_married_lived_separated.example_input'),
```

Comment:
Could you maybe explain what problem this fixes?

Reply:
The render_kw are also set as attributes on the HTML element. Therefore this becomes the `attribute` example_input which is not a valid attribute. By using [a data-attribute](https://developer.mozilla.org/en-US/docs/Learn/HTML/Howto/Use_data_attributes) for that, we get rid of this problem.

Reply:
Nice! Lovin' it!